### ISSUE TITLE
MediaStore sample load photos failed (most blank images)

### ISSUE THREAD

**Issue by skquo** [android/storage-samples#96]
I tested the MediaStore sample on the Redmi Note 9 device, Android Q. Only several photos are loaded, other leave as blank.

![Screenshot_2021-04-30-12-22-50-664_com android samples mediastore](https://user-images.githubusercontent.com/65800210/116652585-b078a680-a9af-11eb-8626-d553defa1202.jpg)

It only works normal with requestLegacyExternalStorage.

**Comment by anik-kucse** [comment]
Check the size for each media object whether it is zero or not. I fixed my problem like that. 

`
val projection = arrayOf(
                MediaStore.Files.FileColumns.SIZE,
                ................................................
            )

....................................................
....................................................

                val fileSizeColumn = cursor.getColumnIndexOrThrow(MediaStore.Files.FileColumns.SIZE)

                while (cursor.moveToNext()) {
                    val fileSize = cursor.getLong(fileSizeColumn)
                    if (fileSize == 0L) {
                        continue
                    }

                   ...............................
                   ........................
                }

`

**Comment by thekingrenz23** [comment]
> Check the size for each media object whether it is zero or not. I fixed my problem like that.
> 
> ` val projection = arrayOf( MediaStore.Files.FileColumns.SIZE, ................................................ )
> 
> .................................................... .................................................... ` ` val fileSizeColumn = cursor.getColumnIndexOrThrow(MediaStore.Files.FileColumns.SIZE)
> 
> ```
>             while (cursor.moveToNext()) {
>                 val fileSize = cursor.getLong(fileSizeColumn)
>                 if (fileSize == 0L) {
>                     continue
>                 }
> 
>                ...............................
>                ........................
>             }
> ```
> 
> `

Yes this will work on your own app but on the gallery it will show those blank images. How can we fix that?
